src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportService.cs

```diff
@@ -97,10 +97,15 @@ public async Task RemoveImportTableRowIfExists(Guid releaseId, string dataFileNa
                 new DatafileImport(releaseId.ToString(), dataFileName));
         }
 
-        public async Task FailImport(Guid releaseId, string dataFileName, IEnumerable<ValidationError> errors)
+        public async Task FailImport(Guid releaseId, string dataFileName, string metaFileName, IEnumerable<ValidationError> errors)
         {
+            var importReleaseMessage =
+                JsonConvert.SerializeObject(BuildMessage(dataFileName, metaFileName, releaseId,
+                    0, 0, ""));
```

Comment:
Why 0 and 0?

Reply:
Seemed like something to put on an error.

The other message used 1 and 1, and I wasn't sure why - I assume they get update later.

I'll set everything to use 1 - I don't think it matters.